src/everything_else/graph.erl

```diff
@@ -180,13 +185,17 @@ update_edge(From, To, Permissions) -> validate([
 -spec remove_edge(From::binary(), To::binary()) -> ok | {error, any()}.
 remove_edge(From, To) -> validate([
         persistence:del(edge_id(From, To)),
-        persistence:set_remove(childrens_id(From), To),
+        persistence:set_remove(children_id(From), To),
         persistence:set_remove(parents_id(To), From)
     ]).
 
 -spec edge_exists(From::binary(), To::binary()) -> {ok, boolean()} | {error, any()}.
 edge_exists(From, To) ->
-    persistence:exists(edge_id(From, To)).
+    case persistence:exists(edge_id(From, To)) of
+        {ok, <<"0">>} -> {ok, false};
+        {ok, <<"1">>} -> {ok, true};
+        {error, Reason} -> {error, Reason}
+    end.
```

Comment:
Przeniósłbym to modułu persystgncji. Jak już ustaliliśmy, funkcja exists aż krzyczy żeby zwracać `{ok, boolean()} | {error, any()}`

Reply:
Yup, się zrobi